src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -43,6 +45,37 @@ private static IVsTextView GetActiveVsTextView()
             return vsTextView;
         }
 
+        public string[] GetCurrentClassifications()
```

Comment:
`GetClassificationsAtCaretPosition`? If you have a selection, this doesn't go grab all of the classifications under the selection or anything like that.